microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -307,6 +307,8 @@ def _kdump_test(self, node: Node, log_path: Path, log: Logger) -> None:
         # is not completed. So we can't use wait_tcp_port_ready function to judge if
         # the dump process is completed and the VM already boots up. We can use
         # try_connect function
+        # Wait for VM boot up and update IP address
+        time.sleep(30)
```

Comment:
Don't hard code sleep in logic. Use longer timeout instead.

Reply:
Have deleted this. I am thinking about a better way to enhance the kdump tests. I will post another PR.